delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/Files.scala

```diff
@@ -700,6 +624,7 @@ object Files {
       case Some(s) if s.rev != c.rev               => IO.raiseError(IncorrectRev(c.rev, s.rev))
       case Some(s) if s.deprecated                 => IO.raiseError(FileIsDeprecated(c.id))
       case Some(s) if s.attributes.digest.computed => IO.raiseError(DigestAlreadyComputed(s.id))
+      case Some(s) if !c.digest.computed           => IO.raiseError(DigestNotComputed(s.id))
```

Comment:
It feels like the issue would have not occurred in the first place if the digest fields in the attribute was an Option maybe?